test/Core.Test/Services/StripePaymentServiceTests.cs

```diff
@@ -155,22 +155,23 @@ public async void PurchaseOrganizationAsync_Stripe_PM(SutProvider<StripePaymentS
             CurrentPeriodEnd = DateTime.Today.AddDays(10),
         });
 
-        var result = await sutProvider.Sut.PurchaseOrganizationAsync(organization, PaymentMethodType.Card, paymentToken, plan, 0, 0, false, taxInfo);
+        var result = await sutProvider.Sut.PurchaseOrganizationAsync(organization, PaymentMethodType.Card, paymentToken, plans, 0, 0
+            , false, taxInfo, false, 8, 10);
 
         Assert.Null(result);
         Assert.Equal(GatewayType.Stripe, organization.Gateway);
         Assert.Equal("C-1", organization.GatewayCustomerId);
         Assert.Equal("S-1", organization.GatewaySubscriptionId);
         Assert.True(organization.Enabled);
         Assert.Equal(DateTime.Today.AddDays(10), organization.ExpirationDate);
-
+        var res = organization.SubscriberName();
         await stripeAdapter.Received().CustomerCreateAsync(Arg.Is<Stripe.CustomerCreateOptions>(c =>
             c.Description == organization.BusinessName &&
             c.Email == organization.BillingEmail &&
-            c.Source == null &&
-            c.PaymentMethod == paymentToken &&
+            c.Source == paymentToken &&
+            c.PaymentMethod == null &&
```

Comment:
Why are PaymentMethod and DefaultPaymentMethod null?

Reply:
this is an existing test. Add a parameter for the secrets manager to ensure it still passes. But if the payment token does not start with pm_, the payment method is null.